### PR TITLE
Update CONTRIBUTING.md  --added steps to contribute

### PR DESCRIPTION
**Pull request by pranavpatel3012**
I have added the "steps to contribute" in file contribute.md.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,4 +17,43 @@ This project and everyone participating in it is governed by the code of conduct
...
+## Steps to contribute
+
+* Comment on the issue you want to work on. Make sure it's not assigned to someone else.
+
+* If you think you encounter a bug or have a suggestion for improvement of code or to add a feature, then create a issue with proper title and tags but first make sure it's not already present. And if you want to work on that, then comment on that issue and wait for the owner to assign that work to you.
+
+### Making a PR
+
+> - Make sure you have been assigned the issue to which you are making a PR.
+> - If you make PR before being assigned, It will be labeled `invalid` and closed without merging.
```
Comment (t0xic0der) [inline CONTRIBUTING.md:31]: I am so glad that you mentioned this. It has been nothing but a headache for us maintainers to deal with folks who directly jump into making PRs without assignment. This is certainly going to come handy.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -17,4 +17,43 @@ This project and everyone participating in it is governed by the code of conduct
...
+    ```
+    git commit -m "Meaningful commit message"
+    ```
+* Push the changes for review
+    ```
+    git push origin <branch-name>
+    ```
+* Create a PR from your repo on Github. Give proper title and description of the changes you made.
+
+* If all goes well, your request will be approved and will be merged.
```
Comment (t0xic0der) [inline CONTRIBUTING.md:59]: Reviewing is also the part of the process. So please add that.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -17,4 +17,43 @@ This project and everyone participating in it is governed by the code of conduct
...
+    ```
+* Commit all the changes
+    ```
+    git commit -m "Meaningful commit message"
+    ```
+* Push the changes for review
+    ```
+    git push origin <branch-name>
+    ```
+* Create a PR from your repo on Github. Give proper title and description of the changes you made.
```
Comment (t0xic0der) [inline CONTRIBUTING.md:57]: Change `Github` to `GitHub`.

Comment (t0xic0der) [reply CONTRIBUTING.md:57]: Change `Give proper title and description of the changes you made.` to `Give a proper title and description for the changes you made.`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -17,4 +17,43 @@ This project and everyone participating in it is governed by the code of conduct
...
+* If you think you encounter a bug or have a suggestion for improvement of code or to add a feature, then create a issue with proper title and tags but first make sure it's not already present. And if you want to work on that, then comment on that issue and wait for the owner to assign that work to you.
+
+### Making a PR
+
+> - Make sure you have been assigned the issue to which you are making a PR.
+> - If you make PR before being assigned, It will be labeled `invalid` and closed without merging.
+
+* Fork the repo. This will create a copy of the current repository that you can edit and make changes to.
+
+* Clone it on your machine. But first navigate to your forked repo and then, there you will find link provided by github to clone.
```
Comment (t0xic0der) [inline CONTRIBUTING.md:35]: Change `github` to `GitHub`.

**Review by t0xic0der — CHANGES_REQUESTED** [review]
Looks good so far. There are some changes I suggested you to make.

**Review by t0xic0der — APPROVED** [review]
Looks good to me.

**Comment by pranavpatel3012** [comment]
> Looks good to me.

Thank you for giving me chance to contribute, glad you liked my contribution.

**Pull request merged**